Source/WarpXUtil.cpp

```diff
@@ -38,9 +38,8 @@ void ReadBoostedFrameParameters(Real& gamma_boost, Real& beta_boost,
 
 void ConvertLabParamsToBoost()
 {
-
-    Real gamma_boost, beta_boost;
-    int max_level;
+    Real gamma_boost = 1., beta_boost = 0.;
+    int max_level = 0;
```

Comment:
valgrind flags these as used without being initialized. are the values I chose the right ones?

Reply:
Yes, these are the right values.